### PR TITLE
Assignment8

### PR DESCRIPTION
**Pull request by AaronSauther**

### CODE REVIEW

**Comment by JerettLatimer** [comment]
Will review!

File: SecretSanta/src/SecretSanta.Web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class HomeController : Controller
         // GET: /<controller>/
         public IActionResult Index()
         {
-            return RedirectToAction("ListGifts", "Gifts");
+            return View();
```
Comment (JerettLatimer) [inline SecretSanta/src/SecretSanta.Web/Controllers/HomeController.cs]: Changed default page back to home.

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -16,20 +16,27 @@
...
 </head>
 <body>
-    <nav>
-        <ul>
-            <li><a asp-controller="Home" asp-action="Index">Home</a></li>
-            <li><a asp-controller="Users" asp-action="Index">Users</a></li>
-            <li><a asp-controller="Groups" asp-action="Index">Groups</a></li>
-            <li><a asp-controller="Gifts" asp-action="Index">Gifts</a></li>
-        </ul>
+    <nav class="navbar" role="navigation">
```
Comment (JerettLatimer) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml]: Fixed navigation.

File: SecretSanta/src/SecretSanta.Web/styles/site.scss [hunk 3]
```diff
@@ -1,3 +1,29 @@
 ﻿@charset "utf-8";
 
+//Set Brand Colors
```
Comment (JerettLatimer) [inline SecretSanta/src/SecretSanta.Web/styles/site.scss]: Good job on your styling!

**Comment by JerettLatimer** [comment]
Good job, it looks like you hit all the requirements!

File: SecretSanta/src/SecretSanta.Web/scripts/components/Gift/giftDetailsComponent.vue [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        }
+
+        @Emit("gift-saved")
+        async saveGift() {
+            let giftClient = new GiftClient();
+            if (this.clonedGift.id > 0) {
+                await giftClient.put(this.clonedGift.id, this.clonedGift);
+            }
+            else {
+                this.clonedGift.userId = Number(this.clonedGift.userId);
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/scripts/components/Gift/giftDetailsComponent.vue]: nice way to do the conversion here as well for number.

**Pull request closed without merging**